src/store/metadata/stub/content/editions/versions/additions-to-the-housing-stock_2022-2023.json

```diff
@@ -0,0 +1,125 @@
+            "download_url": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+          "columns": [
+            {
+                "name": "Period",
+                "datatype": "string",
+                "titles": "Period",
+                "description": "The time period associated with the observation."
+              },
+              {
+                "name": "AREACD",
+                "datatype": "string",
+                "titles": "Statistical Geography",
+                "description": "The geography code associated with the observation."
+              },
+              {
+                "name": "AREANM",
+                "datatype": "string",
+                "titles": "Area Name",
+                "description": "The name assiciated with The geography code "
+              },
+              {
+                "name": "Geography",
+                "datatype": "string",
+                "titles": "Geography Type",
+                "description": "The categorisation of the geography code"
+              },
+              {
+                "name": "Variable Name",
+                "datatype": "string",
+                "titles": "Variable Name",
+                "description": "Which ESS variable is being measured"
+              },
+              {
+                "name": "Indicator", 		
+                "datatype": "string",
+                "titles": "Indicator",
+                "description": "Which ESS indicator is being measured"
+              },
+              {
+                "name": "Mission",
+                "datatype": "string",
+                "titles": "Mission",
+                "description": "Which ESS mission number this dataset is fulfilling"
+              },
+              {
+                "name": "Category",
+                "datatype": "string",
+                "titles": "Category",
+                "description": "Which ESS category the dataset belongs to"
+              },
+              {
+                  "name": "Observation",
+                  "datatype": "integer",
+                  "titles": "Observation",
+                  "description": "Observation value for the chosen variable"
+                },
+                {
+                    "name": "Measure",
+                    "datatype": "string",
+                    "titles": "Measure Type",
+                    "description": "Measure Type of the Observational Value"
+                },
+                {
+                    "name": "Unit",
+                    "datatype": "string",
+                    "titles": "Unit",
+                    "description": "Unit Type of the Observational Value"
+                  },
+                {
+                "name": "Lower Confidence Interval (95%)",
```

Comment:
should be _snake_case_, like in `apprenticeships-achievements_2023-05.json`

`"name": "lower_confidence_interval_95",`